SystemServices/SystemServices.cpp

```diff
@@ -2306,7 +2309,7 @@ namespace WPEFramework {
 								error["message"] = "Invalid region";
 								response["error"] = error;
 								LOGWARN("Please enter valid region");
-								returnResponse(resp);
+								return false;
```

Comment:
please remove. this method doesn't return boolean

Reply:
reverted